packages/@aws-cdk/cloudformation-include/test/test-templates/fn-sub-valid-map.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Resources": {
+    "Bucket": {
+      "Type": "AWS::S3::Bucket",
+      "Properties": {
+        "BucketName": { "Fn::Sub": [ "${Region}-foo-${!Immediate}-foo-${AnotherBucket}-${AnotherBucket.DomainName}-${Name}", 
```

Comment:
Can you please indent this in a standard way? This is virtually unreadable right now.

```json
        "BucketName": {
          "Fn::Sub": [
            "${Region}-foo-${!Immediate}-foo-${AnotherBucket}-${AnotherBucket.DomainName}-${Name}", 
            {
              "Name": { "Ref" : "AnotherBucket" },
              "Region": { "Fn::Base64": "AWS::Region" }
            }
          ]
        }
```
```